website/docs/integrations/github-actions/component-updater.md

```diff
@@ -27,34 +27,88 @@ Discover more details and a comprehensive list of `inputs` and `outputs` in the
+        with:
+          app-id: ${{ secrets.ATMOS_APP_ID }}
+          private-key: ${{ secrets.ATMOS_PRIVATE_KEY }}
+
+      - name: Update Atmos Components
+        uses: cloudposse/github-action-atmos-component-updater@v2
+        env:
+          # https://atmos.tools/cli/configuration/#environment-variables
+          ATMOS_CLI_CONFIG_PATH: ${{ github.workspace }}/rootfs/usr/local/etc/atmos/
+        with:
+          github-access-token: ${{ steps.generate-token.outputs.token }}
+          log-level: INFO
+          vendoring-enabled: true
+          max-number-of-prs: 10
+          include: |
+            aws-*
+            eks/*
+            bastion
+          exclude: aws-sso,aws-saml
+
+      - name: Delete abandoned update branches
+        uses: phpdocker-io/github-actions-delete-abandoned-branches@v2
+        with:
+          github_token: ${{ steps.generate-token.outputs.token }}
+          last_commit_age_days: 0
+          allowed_prefixes: "component-update/"
+          dry_run: no
 ```
 
+### Using a GitHub App
+
+You may notice that we pass a generated token from a GitHub App to `github-access-token` instead of using the native `GITHUB_TOKEN`. We do this because Pull Requests will only trigger GitHub Workflows if the Pull Request is created by a GitHub App or PAT. For reference, see [Triggering a workflow from a workflow](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow).
+
+![Atmos Component Updater GitHub App](/img/github-actions/github-app.png)
+
+To set up a GitHub App for this integration, either install the Cloud Posse managed GitHub App or create an app yourself. Install the Cloud Posse managed app from [github.com/apps/atmos-component-updater](https://github.com/apps/atmos-component-updater). If you wish to instead create the GitHub App yourself, assign only the following Repository permissions:
+
+```diff
++ Contents: Read and write
++ Pull Requests: Read and write
++ Metadata: Read-only
+```
+
+### Using GitHub Environments
+
+We recommend creating a new GitHub environment for Atmos. With environments, the Atmos Component Updater workflow will be required to follow any branch protection rules before running or accessing the environment's secrets. Plus, GitHub natively organizes these Deployments separately in the GitHub UI.
+
+1. Open "Settings" for your repository
+1. Navigate to "Environments"
+1. Select "New environment"
+1. Name the new environment, "atmos".
+1. In the drop-down next to "Deployment branches and tags", select "Protected branches only"
+1. In "Environment secrets", create the two required secrets for App ID and App Private Key from [Using a GitHub App](#using-a-github-app)
+
+Now the Atmos Component Updater workflow will create a new Deployment environment on the next workflow run, easily accessible from the GitHub UI.
+
+![Example Environment](/img/github-actions/github-deployment-environment.png)
```

Comment:
wait for https://github.com/cloudposse/infra-test/pull/670 to be merged and then will take a screenshot of deployments